docs/api-fields.md

```diff
@@ -109,6 +109,8 @@ A checkbox.
 
 ```tsx
 import { BoolField } from 'uniforms-unstyled';
+import { useRef } from 'react'
+const ref = useRef()
```

Comment:
1. Let's keep an empty line between the imports and variables.
2. Add semicolons (Prettier should do that...?).
3. I'd call it `inputRef` here (and `formRef` accordingly).

Reply:
Huh, prettier does fix it, but it also adds `;` at the end of the `tsx`

e.g.
```tsx
<AutoField name="firstName" inputRef={inputRef} />;
```

Reply:
I think it's totally fine.